google/cloud/ndb/context.py

```diff
@@ -625,8 +625,19 @@ def __init__(self, *args, **kwargs):
 
 
 class TransactionOptions(object):
-    def __init__(self, *args, **kwargs):
-        raise exceptions.NoLongerImplementedError()
+    NESTED = 1  # join=False
```

Comment:
Is the integer useful here, since it looks you just use it to lookup a string to then call the appropriate handler? I understand it may have been in the original, but if it's only being used to get a string, maybe we could just use the strings directly? I might have missed something, though.

Reply:
My only reason for keeping the integer was the thought that "_what if_" somebody somewhere had used `@ndb.transactional(propagation=2)` (or similar) in their legacy `ndb` code.

You are 100% right that here in the new `ndb` I only ever lookup a string but just on the off chance some old code has hard coded integers, would we want to break it here or not?

I'm happy either way and will defer to your guidance.

Reply:
That's a reasonable point. It's fine as-is.